src/main/java/com/duckblade/osrs/sailing/module/SailingModule.java

```diff
@@ -48,6 +49,7 @@ Set<PluginLifecycleComponent> lifecycleComponents(
 		LuffOverlay luffOverlay,
 		PrioritizeCargoHold prioritizeCargoHold,
 		RapidsOverlay rapidsOverlay,
+		LightningCloudsOverlay lightningCloudsOverlay,
```

Comment:
not alphabetic ordering

Reply:
I didn't notice that these were in alphabetical order. I thought there were grouped by package. My apologies. I'll know for the future now.